contents/node_operator.rst

```diff
@@ -412,6 +412,101 @@ If the external node is running on IP 12.34.56.78 port 8545, the command would b
 
 You can also use ``https``. But keep in mind that Swarm *does not validate the certificate*.
 
+Connect to the SWAP-enabled testnet
+===================================
+The Swarm project now runs a SWAP-enabled (incentivized) testnet. It uses the same binary as the normal network node, but has incentivization switched on. 
+
+.. important::
+
+  The testnet is highly experimental and may be updated continuously until we release a mainnet version. All tokens used on this testnet are fictitious. Do NOT use this testnet with real value tokens. Funds can be lost.
+
+ 
+The first public incentives-enabled Swarm network runs on the ropsten network. 
+It runs with network id **5**
+
+Prerequisites
+-----------------
+* Create an account on **ropsten**
+* Fund your newly created account with test coins
+
+Network setup
+-------------
+We provide on the public swap-enabled network currently 30 nodes in a kubernetes cluster. 
+
+Bootnodes
+--------------
+
+The bootnodes for the public swap-enabled Swarm cluster run in a separate VMs. 
+The addresses are 
+
+   .. code-block:: none
+
+      enode://7f4d606c91d50d91fd09cb44f8b3d8033f1ca87e977a881e91d77ff6af98b6a52245ba9aeba13a39024ae8bdf3afa421fd018571ae37928c065d7a62503f17a6@3.122.203.99:40301
+      enode://7f4d606c91d50d91fd09cb44f8b3d8033f1ca87e977a881e91d77ff6af98b6a52245ba9aeba13a39024ae8bdf3afa421fd018571ae37928c065d7a62503f17a6@3.122.203.99:40301
+
+Add these addresses with the `--bootnodes` when starting your node.
+
+
+Run your own swap-enabled node and connect to the cluster
+---------------------------------------------------------
+
+All SWAP-related configuration option start with the prefix `swap`. Check the configuration chapter below to consult available options.
+To enable a node to run with the incentivized layer switched on, add the `--swap` flag. However, the `--swap-backend-url` should also be provided. This flag tells the node to which blockchain it will connect and through which provider (e.g. a `geth` node or via `infura`). 
+
+**Get Tokens**
+Getting tokens via the [faucet](https://ropsten.etherscan.io/address/0x49bf80bdee2684580966e476aee0dc3d773ffaf5#writeContract) is only possible *after* your start Swarm. To get tokens, first start Swarm, note down the address of the chequebook and call the `drip` function via the interface of etherscan, with the address of your chequebook as argument.
+
+Note: the current faucet only allows to call the drip function one time per deployed chequebook contract. 
+
+**Start Swarm**
+Note: replace the `bzzkeyhex` with the private key of your funded `bzz-account` and replace the `--datadir` flag with your desired data directory for Swarm.
```

Comment:
I don't think we should tell users to use `bzzkeyhex`, this is only used in the test guide for convenience. "Normal" users should use the regular keystore accounts.